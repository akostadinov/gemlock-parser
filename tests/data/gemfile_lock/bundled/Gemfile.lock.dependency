GEM
  remote: https://rubygems.org/
  specs:
    ast (2.4.2)

PLATFORMS
  ruby

DEPENDENCIES
  ast
  bundler (= 2.0.1)

BUNDLED WITH
   2.0.1
