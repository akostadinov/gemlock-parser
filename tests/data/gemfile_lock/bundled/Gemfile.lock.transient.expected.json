[
  {
    "name": "license_finder",
    "version": "7.1.0",
    "platform": "ruby",
    "pinned": false,
    "remote": "https://rubygems.org/",
    "type": "GEM",
    "path": null,
    "revision": null,
    "ref": null,
    "branch": null,
    "submodules": null,
    "tag": null,
    "requirements": [],
    "dependencies": {
      "license_finder@7.1.0": {
        "bundler@2.0.1": {},
        "rubyzip@2.3.2": {},
        "thor@1.2.1": {},
        "tomlrb@2.0.3": {},
        "with_env@1.1.0": {},
        "xml-simple@1.1.9": {
          "rexml@3.2.5": {}
        }
      }
    }
  },
  {
    "name": "bundler",
    "version": "2.0.1",
    "platform": null,
    "pinned": false,
    "remote": null,
    "type": "GEM",
    "path": null,
    "revision": null,
    "ref": null,
    "branch": null,
    "submodules": null,
    "tag": null,
    "requirements": [],
    "dependencies": {
      "bundler@2.0.1": {}
    }
  },
  {
    "name": "rubyzip",
    "version": "2.3.2",
    "platform": "ruby",
    "pinned": false,
    "remote": "https://rubygems.org/",
    "type": "GEM",
    "path": null,
    "revision": null,
    "ref": null,
    "branch": null,
    "submodules": null,
    "tag": null,
    "requirements": [
      ">= 1",
      "< 3"
    ],
    "dependencies": {
      "rubyzip@2.3.2": {}
    }
  },
  {
    "name": "thor",
    "version": "1.2.1",
    "platform": "ruby",
    "pinned": false,
    "remote": "https://rubygems.org/",
    "type": "GEM",
    "path": null,
    "revision": null,
    "ref": null,
    "branch": null,
    "submodules": null,
    "tag": null,
    "requirements": [
      "~> 1.2"
    ],
    "dependencies": {
      "thor@1.2.1": {}
    }
  },
  {
    "name": "tomlrb",
    "version": "2.0.3",
    "platform": "ruby",
    "pinned": false,
    "remote": "https://rubygems.org/",
    "type": "GEM",
    "path": null,
    "revision": null,
    "ref": null,
    "branch": null,
    "submodules": null,
    "tag": null,
    "requirements": [
      ">= 1.3",
      "< 2.1"
    ],
    "dependencies": {
      "tomlrb@2.0.3": {}
    }
  },
  {
    "name": "with_env",
    "version": "1.1.0",
    "platform": "ruby",
    "pinned": false,
    "remote": "https://rubygems.org/",
    "type": "GEM",
    "path": null,
    "revision": null,
    "ref": null,
    "branch": null,
    "submodules": null,
    "tag": null,
    "requirements": [
      "= 1.1.0"
    ],
    "dependencies": {
      "with_env@1.1.0": {}
    }
  },
  {
    "name": "xml-simple",
    "version": "1.1.9",
    "platform": "ruby",
    "pinned": false,
    "remote": "https://rubygems.org/",
    "type": "GEM",
    "path": null,
    "revision": null,
    "ref": null,
    "branch": null,
    "submodules": null,
    "tag": null,
    "requirements": [
      "~> 1.1.9"
    ],
    "dependencies": {
      "xml-simple@1.1.9": {
        "rexml@3.2.5": {}
      }
    }
  },
  {
    "name": "rexml",
    "version": "3.2.5",
    "platform": "ruby",
    "pinned": false,
    "remote": "https://rubygems.org/",
    "type": "GEM",
    "path": null,
    "revision": null,
    "ref": null,
    "branch": null,
    "submodules": null,
    "tag": null,
    "requirements": [],
    "dependencies": {
      "rexml@3.2.5": {}
    }
  }
]
